dev-tools/reconfigurator-cli/tests/input/cmds-mupdate-update-flow.txt

```diff
@@ -124,30 +124,42 @@ blueprint-diff latest
 sled-set serial1 mupdate-override unset
 inventory-generate
 
-# This will attempt to update the first sled's host OS. Walk through that update
-# and the host OS of the two other sleds.
+# This will attempt to update the RoT bootloader on the first sled.
```

Comment:
This is kind of followup to https://github.com/oxidecomputer/omicron/pull/8832#discussion_r2283153789. On that PR, I had to change this test to update the host OS so that we could move on to the planner doing things with zones, but in review, the question came up of "why don't we also have to do the same thing for the bootloader, RoT, and SP?". The superficial answer was "the fake TUF repo doesn't have any matching artifacts for those so the planner is skipping them", but I didn't dig into why that is. With the changes on this PR, the fake TUF repo now _does_ have matching artifacts. So instead of just updating the host OS here, we have to do all the MGS-based updates for all three fake sleds.